gcampus/core/models/measurement.py

```diff
@@ -254,3 +253,28 @@ def __str__(self):
             "pk": self.pk,
             "name": self.parameter_type.name,
         }
+
+
+class Calibration(models.Model):
```

Comment:
The class `"Calibration"` is missing from the `__all__` variable at then beginning of the file. While this does not raise an error, `__all__` is used to tell python linters and your IDE what classes can be imported from this file.

Reply:
fixed in 067baa0b47adfd6587eff8d377fe351906522832